src/Jawa/Event/Pause.elm

```diff
@@ -0,0 +1,47 @@
+module Jawa.Event.Pause exposing (Pause, decoder, encoder)
+
+{-|
+
+@docs Pause, decoder, encoder
+
+-}
+
+import Jawa.Viewable as V
+import Json.Decode
+import Json.Encode
+
+
+{-| <https://docs.jwplayer.com/players/reference/playback-events-1#onpause>
+-}
+type alias Pause =
+    { newstate : String
+    , oldstate : String
+    , pauseReason : String
+    , reason : String
```

Comment:
I suspect that these states and reasons are actually enums rather than strings. 

https://github.com/jwplayer/jwplayer/blob/c280de15ff78927c026010e4e7a83054e4638240/src/js/events/events.ts#L7

https://github.com/jwplayer/jwplayer/blob/c280de15ff78927c026010e4e7a83054e4638240/src/js/controller/model.ts#L17